lockapi.pm

```diff
@@ -29,9 +29,16 @@ sub _lock_action {
     my ($name, $where) = @_;
     my $param = {action => 'lock'};
     $param->{where} = $where if $where;
-    my $res = api_call('post', "mutex/$name", $param)->code;
-    return 1 if ($res == 200);
-
+    # try up to 7 times
+    my $res = "";
+  retry:
```

Comment:
The `retry` label is indeed not used here and what's worse IMO is that the `retry` label is used twice. How perl should know which one to use?
But I have issue even with what is the patch trying to solve. `barrier_wait` and `mutex_lock` both waits already, for `*_try_*` function I thing we don't want to wait anyway. In the mentioned poo# I really didn't get why is this needed.